container/root/run.d/08-php-opcache.sh

```diff
@@ -1,9 +1,8 @@
-#!/usr/bin/with-contenv bash
+#!/bin/bash
```

Comment:
@ko-be yes, from the notes: 
```Run.d: moved new relic and opcache (debug) scripts
To account for non-S6 workloads like workers and crons, move startup mechanisms to run.d instead of S6.```